### ISSUE TITLE
Categorize activities

### ISSUE THREAD

**Issue by Rafael94** [elsa-workflows/elsa-core#1350]
Hi,

in the `ActivityAttribute` attribute there is a `Traits` property (`Elsa.Metadata.ActivityTraits`). 

I would like to use this to sort out `WorkflowExecutionLogRecord` entries. For example `If, Loop, Switch` don't interest the user, he just wants to see the actions (folder created, email sent, ...).

What do you think about adding `ControlFlow` and `ManualTrigger` (UserTask) in `ActivityTraits` and categorize the existing activities. 

For me, it would be best if `ActivityTraits` is stored in `WorkflowExecutionLogRecord` as well. Then I could filter on it directly and the data doesn't have to be loaded from the database.

**Comment by sfmskywalker** [comment]
Makes sense. Then the Workflow Journal UI can be updated separately with filters too. Do you want to do a PR?

**Comment by Rafael94** [comment]
Should I then add to `ActivityTraits` the `ControlFlow`,`ManualTrigger` and `AutoTrigger` (Timer) entries or do you have better names for them?

**Comment by sfmskywalker** [comment]
I think that's fine. Alternatively, we could consider splitting `ManualTrigger` and `AutoTrigger` into these:

- Trigger
- Manual
- Auto

Traits can be logically ORed, so one activity could be `Trigger+Manual` and another `Trigger+Auto`.

On the other hand, it makes me wonder what it would mean for an activity to just have `Auto` or `Manual`. Perhaps that would be more useful as "sub traits", but that might be taking it too far.

If the filter supports multiple traits to be selected, than having `ManualTrigger` and `AutoTrigger` might be fine to start with.
